CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Ivanchuk-Beta2
+## Unreleased as of Sprint 118 - 2019-08-19
+
+### Fixed
```

Comment:
I think this should be `Added`.

Reply:
hmmm,...that PR is labeled as a bug (Fixed). But I can change it to Added.

Reply:
Already discussed with @JPrause but for future reference... Adding those modules was a part of a bug fix. But it just looked strange to put that PR in `Fixed` section when there is no "issue" mentioned and title says "Install modules"...